### PR TITLE
Build each subproject, not just rholang, rosette

### PR DESCRIPTION
**Pull request by dckc**
In the Nov 3 "CI configuring notes" design, only directories changed
in the last commit are built. But if a PR has a number of commits, the
last of which is only a change to the top-level README, any
subdirectories wouldn't be re-built. Some searching suggests this is
an unsolved problem:

  - https://stackoverflow.com/questions/39596177/monorepo-travis-and-matrix-builds-how-do-i-build-a-specific-path-only-when-it
  - https://github.com/travis-ci/travis-ci/issues/6301

So let's just build everything.

### CODE REVIEW

**Comment by dckc** [comment]
I'm blocked by `not found: type RChainException` in comm #58

**Comment by dckc** [comment]
5 different scala versions (#57) and 2 different sbt versions (#55) don't help either.

**Comment by DoSOfRedRiver** [comment]
Can we track changed files with $TRAVIS_COMMIT_RANGE? 
` The range of commits that were included in the push or pull request. (Note that this is empty for builds`.
https://docs.travis-ci.com/user/environment-variables/

**Comment by dckc** [comment]
See #60 where I rebased against dev.

File: rosette/test.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Modify configuration.rbl to run tests
```
Comment (dckc) [inline rosette/test.sh]: modifying checked-in files is... messy

**Pull request closed without merging**